connector/mqtt/vernemq/v2k-bridge/app/config.js

```diff
@@ -1,74 +1,69 @@
 const { toBoolean } = require('./utils');
 
 const app = {
-  baseDir: process.env.BASE_DIR || '/opt/v2k_bridge',
-  hostname: process.env.HOSTNAME || 'v2k-bridge',
-  logger: {
-    transports: {
-      console: {
-        level: process.env.V2K_CONSOLE_LOG_LEVEL || 'info',
-      },
-    },
-    verbose: (toBoolean(process.env.V2K_LOG_VERBOSE)) || false,
-  },
+  basedir: process.env.V2K_APP_BASEDIR,
+  'connection.retry.count': parseInt(process.env.V2K_APP_CONNECTION_RETRY_COUNT, 0),
```

Comment:
you're not forcing base 10. Is the expected behavior here for the parseInt()?

Reply:
The behaviour is correct, since you can pass numbers between 2 and 36 and anything outside this scope is treated as 10, but I'll change this because it's ugly.